cli/cmd/vulnerability.go

```diff
@@ -353,23 +353,28 @@ func buildVulnContainerAssessmentReports(assessment *api.VulnContainerAssessment
 		Packages:             filterVulnContainerImagePackages(assessment.Image),
 	}
 
-	if cli.JSONOutput() {
+	switch {
+	case cli.JSONOutput():
 		filteredAssessment := assessment
 		filteredAssessment.Image.ImageLayers = details.VulnerabilityDetails.ImageLayers
 		if err := cli.OutputJSON(filteredAssessment); err != nil {
 			return err
 		}
-	} else {
+	case cli.CSVOutput():
+		if err := cli.OutputCSV(buildVulnerabilityDetailsReportCSV(details)); err != nil {
+			return err
+		}
```

Comment:
For these switch statements, what happens if more than one case statement is true? to they execute in order? so if someone here passes `--cvs --json` we will display ONLY Json output? 🤔

Reply:
Yep; that's right.  These execute in order. 

For that situation specifically, we should probably look for conflicting output formats and error out; but as the code exists today a customer would get a different result than they might expect based on the order.